workflow/tests/EPA_Tests/MF_National_1.2/MFNCv12_CZ2_FL_gas_ground_corner_slab.xml

```diff
@@ -381,47 +381,47 @@
         <LightingGroup>
           <SystemIdentifier id='LightingGroup1'/>
           <Location>interior</Location>
-          <FractionofUnitsInLocation>0.0</FractionofUnitsInLocation>
+          <FractionofUnitsInLocation>1.0</FractionofUnitsInLocation>
```

Comment:
These changes should be made via tasks.rb, not manually. They will be overridden the next time someone runs `tasks.rb update_hpxmls`. The CI is supposed to have a check for this but had a bug so it didn't catch it. I just fixed the CI check (I think), so it will probably throw an error now until you fix it.